subsys/bluetooth/controller/Kconfig

```
@@ -547,13 +547,12 @@ config BT_CTLR_SDC_CIS_SUBEVENT_LENGTH_US
 	  is chosen by the controller.
 
 config BT_CTLR_CHANNEL_SOUNDING
-	bool "Channel Sounding support [EXPERIMENTAL]"
-	select EXPERIMENTAL
+	bool "Channel Sounding support"
 	help
 	  Enable Channel Sounding support.
 
 config BT_CTLR_SDC_CS_COUNT
-	int "Number of concurrent connections supporting CS procedure [EXPERIMENTAL]"
+	int "Number of concurrent connections supporting CS procedure"
```

```suggestion
	int "Number of concurrent connections supporting CS procedures"
```

maybe a good opportunity to fix the language here